### PR TITLE
Check and fix cacheMaxAge

### LINKED ISSUE #35: Check and fix cacheMaxAge
set the max time before clients check for new data to 23 hours for forklifted items (ie, in metatables), as forklift updates at most once every day.

```py
manager = arcgis.features.FeatureLayerCollection.fromitem(item).manager
manager.update_definition({'cacheMaxAge': 82800}) #: 23 hours of seconds
```

### PR DESCRIPTION
**Pull request by jacobdadams**
Checks and fixes the cacheMaxAge property for any AGOL hosted feature service in the SGID. Value (in seconds) is specified in the credentials file. 

Closes #35

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 setup(
     name='auditor',
-    version='1.3.0',
+    version='1.4.0',
```
Comment (steveoh) [inline setup.py:14]: **suggestion**: bumping versions in branches can get a little funky when features etc happen asynchronously or on top of each other. It is simpler to bump versions only on master.

**Review by steveoh — APPROVED** [review]
looks great!

File: setup.py (hunk 1 above)
Comment (jacobdadams) [reply setup.py:14]: Yeah, I've been seeing how that can get tricky, including having the version increment buried in a random commit. Is the best procedure to just do all the code updates for the new feature, do the PR, and then after the PR has been merged go do a single commit to increment the version right on master w/o a PR review?

Comment (steveoh) [reply setup.py:14]: you got it.

**Pull request merged**